pkg/cli/build/build.go

```diff
@@ -199,6 +204,36 @@ func isImageDefinitionValid(ctx *image.Context) bool {
 	return true
 }
 
+func downloadKubernetesSELinuxPolicies(ctx *image.Context) error {
+	if ctx.ImageDefinition.Kubernetes.Version == "" {
+		return nil
+	}
+
+	configPath := filepath.Join(ctx.ImageConfigDir, "kubernetes", "config", "server.yaml")
+	config, err := kubernetes.ParseKubernetesConfig(configPath)
+	if err != nil {
+		return fmt.Errorf("parsing kubernetes server config: %w", err)
+	}
+
+	selinuxEnabled, _ := config["selinux"].(bool)
+	if !selinuxEnabled {
+		return nil
+	}
+
+	audit.Audit("SELinux is enabled in the Kubernetes configuration. " +
+		"The necessary RPM packages will be downloaded.")
+	zap.S().Info("SELinux mode requested for Kubernetes")
+
+	rpmDir := filepath.Join(ctx.ImageConfigDir, "rpms")
+	gpgKeysDir := filepath.Join(rpmDir, "gpg-keys")
```

Comment:
Similar point here as above, I don't love the idea of the directory structure being hardcoded in multiple places. Can we make this a function in the rpm file to keep the details of how the config directory is laid out in one location?

Reply:
This one is even worse. 😅 

It is duplicated in both the `combustion` and `rpm/resolver` packages. I'll think about it.